### ISSUE TITLE
How to compile the library as a shared object file (.so)

### ISSUE THREAD

**Issue by frallain** [ermig1979/Simd#36]
I am not well-versed into the compilation process with cmake/make and the current process gives a list of Object files (.o)  i.e.
```bash
git clone https://github.com/ermig1979/Simd.git
cd Simd/prj/cmake/
cmake . -DTOOLCHAIN="" -DTARGET="" -DCMAKE_BUILD_TYPE="Release"
make
``` 
What should be done to link all these object files into one shared object .so file?
Thanks in advance

**Comment by ermig1979** [comment]
cmake . -DTOOLCHAIN="" -DTARGET="" -DCMAKE_BUILD_TYPE="Release" -DLIBRARY="SHARED"

**Comment by frallain** [comment]
fixed with commit  0b24b5ca7f2205b1af1ad75972759a2b312ddd69 on 2017-12-13
